rllib/algorithms/algorithm.py

```diff
@@ -37,6 +37,7 @@
 from ray.rllib.algorithms.algorithm_config import AlgorithmConfig
 from ray.rllib.algorithms.registry import ALGORITHMS_CLASS_TO_NAME as ALL_ALGORITHMS
 from ray.rllib.connectors.agent.obs_preproc import ObsPreprocessorConnector
+from ray.rllib.core.columns import Columns
```

Comment:
Already the PR title made my heart beat faster :D

Reply:
Haha!